.github/actions/maintenance-banner-deploy/action.yml

```diff
@@ -0,0 +1,30 @@
+name: Maintenance banner deploy
+description: Sends an email to the support team when the state of the maintenance banner changes in PROD
```

Comment:
Is this description accurate? The email portion doesn't seem to be included here.

Reply:
It is not accurate! :fire:

Reply:
:heavy_check_mark: